client/signup/steps/design-picker/index.jsx

```diff
@@ -268,9 +267,6 @@ export default function DesignPickerStep( props ) {
 	function renderCategoriesFooter() {
 		return (
 			<>
-				{ useFeaturedPicksButtons && (
-					<FeaturedPicksButtons designs={ featuredPicksDesigns } onSelect={ pickDesign } />
-				) }
```

Comment:
Im not super familiar with this area of the code, but I think instead of removing `FeaturedPicksButtons` we may want to remove blank canvas from the `featuredPicksDesigns` that gets fed to that component.

It looks like there can be other designs that show up in some circumstance that this might get rid of as i no longer see them testing this PR:

![Screen Shot 2022-03-31 at 8 45 13 AM](https://user-images.githubusercontent.com/28742426/161057701-16f343f6-3808-4305-bbef-444d7594ca66.png)

Reply:
thanks for the catch. will update accordingly